biomage/rotate_ci/rotate_ci.py

```diff
@@ -275,10 +278,23 @@ def rotate_ci(token, org):
     repos = exclude_iac_from_rotation(repos, org.login)
 
     policies = [ret for ret in (filter_iam_repos(repo) for repo in repos) if ret]
+    policies = dict(policies)
+
     click.echo(
-        f"Found {len(policies)} repositories marked as requiring CI IAM policies."
+        f"Found {len(policies.keys())} repositories marked as requiring CI IAM policies.\nThese are: {policies.keys()}" 
```

Comment:
I tried this line and it comes out as : 

<img width="537" alt="image" src="https://user-images.githubusercontent.com/2862362/216317394-a8d79cd8-71f9-443c-b43e-699c07fcfde9.png">

With a bit of improvement: `', '.join(policies.keys())`, it will look a bit better:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/2862362/216317727-2f949b24-68d5-4169-8b22-a743f30849c1.png">